packages/VirtualMidiKeyboardNoSound/src/host.js

```diff
@@ -22,12 +22,23 @@ const mountPlugin = (domNode) => {
 };
 
 (async () => {
+	// Init WamEnv
+	const { default: apiVersion } = await import("../../api/src/version.js");
+	const { default: addFunctionModule } = await import("../../sdk/src/addFunctionModule.js");
+	const { default: initializeWamEnv } = await import("../../sdk/src/WamEnv.js");
+	await addFunctionModule(audioContext.audioWorklet, initializeWamEnv, apiVersion);
+	const { default: initializeWamGroup } = await import("../../sdk/src/WamGroup.js");
+	const hostGroupId = 'test-host';
+	const hostGroupKey = performance.now().toString();
+	await addFunctionModule(audioContext.audioWorklet, initializeWamGroup, hostGroupId, hostGroupKey);
```

Comment:
these embedded hosts could all be updated to use the `initializeWamEnv` function added by @Fr0stbyteR